### PR TITLE
[2024-11 CWG Motion 8] P3176R1 The Oxford variadic comma

### PR DESCRIPTION
**Pull request by Eisenwave**
Fixes https://github.com/cplusplus/draft/issues/7412

Fixes https://github.com/cplusplus/papers/issues/1839

### CODE REVIEW

**Comment by jensmaurer** [comment]
Please squash and rebase and use the commit message format as described here:

https://github.com/cplusplus/draft/wiki/Commit-message-format

**Review by jensmaurer — APPROVED** [review]

File: source/declarations.tex [hunk 1]
```diff
@@ -3659,9 +3661,13 @@
...
 part of an \grammarterm{abstract-declarator},
-``\tcode{, ...}''
+``\tcode{...}''
 is synonymous with
-``\tcode{...}''.
+``\tcode{, ...}''.
+A \grammarterm{parameter-declaration-clause}
+of the form
+\grammarterm{parameter-declaration-list} \tcode{...}
+is deprecated.
```
Comment (AlisdairM) [inline source/declarations.tex:3670]: Should we have a iref to [depr.ellipsis.comma]?

Comment (jensmaurer) [reply source/declarations.tex:3670]: sounds good

File: source/future.tex [hunk 2]
```diff
@@ -115,6 +115,19 @@
 \end{codeblock}
 \end{example}
 
+\rSec1[depr.ellipsis.comma]{Non-comma-separated ellipsis parameters}
+
+A \grammarterm{parameter-declaration-clause}
+of the form
+\grammarterm{parameter-declaration-list} \tcode{...}
```
Comment (AlisdairM) [inline source/future.tex:122]: Likewise, I think this should have an iref back to [dcl.fct] (I think that is the right clause)

File: source/declarations.tex (hunk 1 above)
Comment (Eisenwave) [reply source/declarations.tex:3670]: Can we actually do that on this PR? I have been told that any editorial fixes that would change the approved wording should be done as a separate follow-up PR.

Comment (tkoeppe) [reply source/declarations.tex]: Done.

**Pull request merged**